providers.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  version = "~> 2.70"
+}
```

Comment:
Was this needed to make it work? Usually modules don't need providers and they can cause some backwards compatibility issues.

Reply:
I got some strange error re: `Error verifying checksum for provider "aws"` & adding the provider fixed it. Is there a better way?

Reply:
I'm not sure about that error, but may be I can dig around and see what's up. My expectation is that we don't ship providers in modules because you end up in this dependency hell where you can't remove a module without the provider that was also removed. It's a nightmare I dug myself out of once. I'll go play with the module and get back to you though.

Reply:
I made this PR and couldn't reproduce the error above. Can you try it out? https://github.com/trussworks/terraform-aws-ses-domain/pull/52